skbio/sequence/_grammared_sequence.py

```diff
@@ -562,8 +634,8 @@ def expand_degenerates(self):
 
```

Comment:
I can't comment on the relevant line because it's outside the diff, but can you update the *Yields* section above to use *definite*?